fpdcc/ccfp-asset-dashboard
# Fix various bugs + improvements from user feedback

## Overview
Closes:

- #183 
- #174
- #181 
- #175
- #180

## Testing Instructions
- follow readme instructions to load gis data if you don't already have it
- make sure all the money values have no decimal (# 183)
    - this'll be on the project detail page, update phase page, create/update funding stream, cip planner
 - add some assets and make sure you can delete all of them (# 147)
 - export a csv from the cip planner and make sure the csv includes a funding_secured column 
 - duplicate the gis search from (#175) and make sure a message appears
 - for #180
     - save a phase as new and don't input a number into the actual cost field. it should save since we're not validating that field but we're showing it anyway.
     - set a phase as "complete" and try to input this year or older (because some js removes the minimum input validation)
